hazelcast/src/test/java/com/hazelcast/test/starter/ConfigConstructor.java

```diff
@@ -87,18 +82,18 @@ private static Object cloneConfig(Object thisConfigObject, ClassLoader classload
                 continue;
             }
             Class returnType = method.getReturnType();
-            Method setter;
             Class<?> otherReturnType;
             try {
                 otherReturnType = getOtherReturnType(classloader, returnType);
             } catch (ClassNotFoundException e) {
                 // new configuration option, return type was not found in target classloader
-                LOGGER.info(format("Configuration option %s is not available in target classloader: ", method.getName(),
+                debug(format("Configuration option %s is not available in target classloader: %s", method.getName(),
```

Comment:
Is it okay to log this via `debug()`, which is disabled by default? It saves us a logger instance here in favor of a single on in the utils class.